templates/user/account.html.twig

```diff
@@ -1,44 +1,41 @@
-{% extends 'base.html.twig' %}
+{% extends 'user/user_layout.html.twig' %}
 
-{% block body %}
-<div class="container-fluid bg-light py-5">
-    <div class="container">
-        <div class="row">
-            <aside class="col-4">
-                <h1>{{ app.user.name }}</h1>
-                <ul class="nav flex-column nav-pills">
-                    <li class="nav-item">
-                        <a href="#" class="nav-link active">
-                            Future Conferences
-                        </a>
-                    </li>
-                    <li class="nav-item">
-                        <a href="#" class="nav-link">
-                            My Talks
-                        </a>
-                    </li>
-                    <li class="nav-item">
-                        <a href="#" class="nav-link">
-                            My Participations
-                        </a>
-                    </li>
-                    <li class="nav-item">
-                        <a href="#" class="nav-link">
-                            Edit my account
-                        </a>
-                    </li>
-                    <li class="nav-item">
-                        <a href="{{ app.user ? path('user_account') : path('login') }}" class="nav-link">
-                            {{ app.user ? '➡️ Logout' }}
-                        </a>
-                    </li>
-                </ul>
-            </aside>
+{% block content %}
+<aside class="col-4">
+    <h1>{{ app.user.name }}</h1>
+    <ul class="nav flex-column nav-pills">
+        <li class="nav-item">
+            <a href="#" class="nav-link active">
+                Future Conferences
+            </a>
+        </li>
+        <li class="nav-item">
+            <a href="#" class="nav-link">
+                My Talks
+            </a>
+        </li>
+        <li class="nav-item">
+            <a href="{{ path('user_participation') }}" class="nav-link">
+                My Participations
+            </a>
+        </li>
+        <li class="nav-item">
+            <a href="#" class="nav-link">
+                Edit my account
+            </a>
+        </li>
+        <li class="nav-item">
+            <a href="{{ app.user ? path('user_account') : path('login') }}" class="nav-link">
+                {{ app.user ? '➡️ Logout' }}
```

Comment:
I'm not sure about this one, since I have a login text on my local instance, but I didn't look after it. Will do, letting this open for now

Reply:
Where do it come from? 🤔

Reply:
Ok so I didn't read the commentary well the first time. I thought we were talking about the header login button.

We don't have any login text on this button, but since the user must be logged in to access this page, we probably should leave it like this, no ?

Reply:
there is an if (ternary) just above 🤔 I don't understand the code so